src/Facade/Services/WEEEReportsService.cs

```diff
@@ -16,12 +16,16 @@ public WEEEReportsService(IWEEEReports weeeReports)
             this.WEEEReports = weeeReports;
         }
 
-        public IResult<ResultsModel> GetWEEEReport(WEEEReportRequestResource resource)
+        public IResult<ResultsModel> GetUkWeeeReport(DateTime fromDate, DateTime toDate)
         {
-            var results = this.WEEEReports.GetUKWEEEReport(
-                DateTime.Parse(resource.FromDate),
-                DateTime.Parse(resource.ToDate));
+            var results = this.WEEEReports.GetUkWEEEReport(fromDate, toDate);
             return new SuccessResult<ResultsModel>(results);
         }
+
+        public IResult<IEnumerable<ResultsModel>> GetGermanWeeeReport(DateTime fromDate, DateTime toDate)
```

Comment:
Are there tests for these? I'd stick them in personally even though there isn't much in the way of logic yet.